### PR TITLE
use shared GHA workflow

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

**Comment by evgeni** [comment]
Ruby 3.0 will be failing until Katello is fixed, you can ignore that for now

**Review by maccelf — APPROVED** [review]
Thank you, LGTM

File: test/unit/host_managed_extensions_test.rb [hunk 1]
```diff
@@ -18,7 +18,7 @@ def test_update_kernel_version
       package_json = { :name => 'kernel', :version => '1',
                        :release => '1.el7', :arch => 'x86_64', :epoch => '1',
                        :nvra => 'kernel-1-1.el7.x86_64' }
-      host.import_package_profile([::Katello::Pulp::SimplePackage.new(package_json)])
+      host.import_package_profile([::Katello::SimplePackage.new(package_json)])
```
Comment (maccelf) [inline test/unit/host_managed_extensions_test.rb:21]: Do I need to release a new version after that ?

Comment (evgeni) [reply test/unit/host_managed_extensions_test.rb:21]: Only if you want to be compatible with Katello 4.8+ :)

No, seriously, given I had to edit `app/models/concerns/foreman_kernel_care/host_managed_extensions.rb` it indicates that the current release is not compatible with current Katello releases, so I think a release of kernel_care should happen.

That said, kernel_care being broken doesn't block anything (besides k_c itself), so it's totally up to you when you have time for a release :)

Comment (maccelf) [reply test/unit/host_managed_extensions_test.rb:21]: Okay got it. Thanks

**Comment by nadjaheitmann** [comment]
@maccelf I think we should release it sooner than later. I would go for version 2.0.0, any objections? I can take care of the process if you want.

**Comment by maccelf** [comment]
@nadjaheitmann no objections. Yes, it will be nice. Thank you

**Pull request merged**